llama_hub/library.json

```diff
@@ -983,5 +983,14 @@
       "repository",
       "collaborators"
     ]
+  },
+  "NougatOCRReader": {
+    "id": "github_repo_collaborators",
```

Comment:
id should be "nougat_ocr"

Reply:
and NougatOCRReader should match the class name and readme PDFNougatOCR